src/CardContent/Infographic/Infographic.tsx

```diff
@@ -36,7 +36,7 @@ const Infographic = ({ media, version }: InfographicProps) => {
           width: 336
         }}
       >
-        <Box padding={1}>
+        <Box padding={1} sx={{ maxWidth: 320 }}>
```

Comment:
please implement the same suggestion i requested on the other PR

Reply:
Done on the other PR and pulled into this one.